### PR TITLE
add swin unetr placeholder repository

### PR DESCRIPTION
**Pull request by ahatamiz**
This PR adds Swin UNETR placeholder repository. 

Signed-off-by: ahatamizadeh <ahatamizadeh@nvidia.com>

### CODE REVIEW

**Comment by ahatamiz** [comment]
Hi @wyli I have added the placeholder for Swin UNETR research repository. Appreciate it if you can please review this PR. 

Thanks

File: UNETR/BTCV/README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/unetr-transformers-for-3d-medical-image/medical-image-segmentation-on-synapse-multi)](https://paperswithcode.com/sota/medical-image-segmentation-on-synapse-multi?p=unetr-transformers-for-3d-medical-image)
```
Comment (wyli) [inline UNETR/BTCV/README.md:1]: will this badge update automatically if the state-of-the-art method changes?

Comment (ahatamiz) [reply UNETR/BTCV/README.md]: Hi @wyli , It could be left un-updated. Hence, I removed it. Please review the latest PR.

**Review by wyli — APPROVED** [review]
Thanks, it looks good to me.

**Pull request merged**